<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page_devicelist">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_devicelist">
          <property name="font">
           <font>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="title">
           <string>设备列表</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="DeviceList" name="devicelist" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_settings">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="netsettings">
           <attribute name="title">
            <string>网络设置</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_address"/>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_gateway"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_gataway">
              <property name="text">
               <string>网关</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_address">
              <property name="text">
               <string>IPv4地址</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_netmask">
              <property name="text">
               <string>子网掩码</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_netmask">
              <property name="focusPolicy">
               <enum>Qt::TabFocus</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="ScenseList" name="scenselist">
           <attribute name="title">
            <string>广告设置</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>317</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeviceList</class>
   <extends>QWidget</extends>
   <header location="global">devicelist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScenseList</class>
   <extends>QWidget</extends>
   <header>scenselist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
